docs/docs/target/kubernetes.md

```diff
@@ -318,31 +375,31 @@ Scan the cluster for Kubernetes Pod Security Standards Baseline compliance:
 
 ```
 
-$ trivy k8s cluster --compliance=k8s-pss-baseline --report summary
+trivy k8s --compliance=k8s-pss-baseline --report summary
```

Comment:
I like that cluster is no longer needed because it is a given when using `trivy k8s`